django_celery_results/models.py

```diff
@@ -51,7 +51,6 @@ class TaskResult(models.Model):
                     'used with the task'))
     status = models.CharField(
         max_length=50, default=states.PENDING,
-        choices=TASK_STATE_CHOICES,
```

Comment:
I think it is a breaking change. ma I know what benefits removing this offer and what else we can do without removing this? and thanks for handling the issue

Reply:
Celery allows for settings statuses that are not part of `TASK_STATE_CHOICES`, such as `PROGRESS`. However, when set to a non-standard value, `django-celery-results` will fail to show the status, displaying it as `-` instead.

Reply:
I would love to see examples and screenshots if possible

Reply:
as far I can see the states are coming from celery https://github.com/celery/django-celery-results/blob/main/django_celery_results/models.py#L5

Reply:
https://github.com/celery/celery/blob/3b20010020cf810a0ba40a27d6d4e83210d4a5e2/celery/states.py#L103-L104

In the celery/states file, you can see that they allow for "custom states" that are not defined in the file.

Even the documentation suggests this with the `PROGRESS` state: https://docs.celeryq.dev/en/stable/userguide/calling.html#on-message

If a developer followed the above userguide provided by Celery, then DCR will show a `-` instead of `PROGRESS` because of this.